### PR TITLE
replace rinkeby with goerli

### PR DESCRIPTION
**Pull request by auryn-macmillan**
# Description

Rinkeby has been deprecated, so this PR replaces Rinbkeby with Goerli.

### CODE REVIEW

File: packages/app/src/connectors/index.ts [hunk 1]
```diff
@@ -26,7 +25,7 @@ const NETWORK_URLS: { [key in SupportedChainId]: string } = {
 
 export const network = new NetworkConnector({
   urls: NETWORK_URLS,
-  defaultChainId: 4,
+  defaultChainId: 420,
```
Comment (jfschwarz) [inline packages/app/src/connectors/index.ts:28]: Isn't goerli chain ID 5?

File: packages/app/src/services/poster/hooks/usePoster.ts [hunk 2]
```diff
@@ -31,6 +31,8 @@ const usePoster = () => {
   const [loading, setLoading] = useState<boolean>(false)
   const { pinAction } = useFiles()
   const isValidChain = chainId && checkIsValidChain(chainId, network).isValid
+  const parameters = chainParameters(chainId ? chainId : 420)
```
Comment (manboy-eth) [inline packages/app/src/services/poster/hooks/usePoster.ts:34]: Perhaps use
```
  SupportedChain.GOERLI
```

File: packages/app/src/connectors/index.ts (hunk 1 above)
Comment (auryn-macmillan) [reply packages/app/src/connectors/index.ts:28]: ha! My mistake.

File: packages/app/src/services/poster/hooks/usePoster.ts (hunk 2 above)
Comment (auryn-macmillan) [reply packages/app/src/services/poster/hooks/usePoster.ts]: Done

**Review by manboy-eth — APPROVED** [review]
LGTM

**Pull request merged**